packages/shopify-codemod/test/transforms/mocha-context-to-global-references.test.js

```diff
@@ -0,0 +1,28 @@
+import 'test-helper';
+import mochaContextToGlobalReferences from 'mocha-context-to-global-references';
+
+describe('mochaContextToGlobalReferences', () => {
+  it('renames the member expression to use a test global', () => {
+    expect(mochaContextToGlobalReferences).to.transform('mocha-context-to-global-references/rename', {
+      testContextToGlobals: {
+        sinon: {properties: ['spy', 'stub', 'server']},
+      },
+    });
+  });
+
+  it('replaces a contextual reference to a global reference', () => {
+    expect(mochaContextToGlobalReferences).to.transform('mocha-context-to-global-references/replace', {
+      testContextToGlobals: {
+        testClock: {
```

Comment:
Good points, I see where you are coming from, but I think I'm going to stick with what I have here. 1) I think it reads better for when there are multiple properties, particularly when those properties are not being replaced, and 2) I am too lazy to update all the tests :P